beps/bep_0003.rst

```diff
@@ -87,70 +92,115 @@ announce
 
 info
   This maps to a dictionary, with keys described below.
+  
+``piece layer``
```

Comment:
I interpret this to be a list where each entry represents a file, and the content of each layer is the full tree (except truncated at the piece size). I would expect the key to be called something like "merkle trees" or at least plural "piece layers"